addons/crm/security/ir.model.access.csv

```diff
@@ -21,3 +21,5 @@ access_calendar_event_type_sale_manager,calendar.event.type.manager,calendar.mod
 access_calendar_event_type_sale_user,calendar.event.type.user,calendar.model_calendar_event_type,base.group_user,1,0,0,0
 access_calendar_event_type_sale_salesman,calendar.event.type.salesman,calendar.model_calendar_event_type,sales_team.group_sale_salesman,1,0,0,0
 access_mail_activity_type_sale_manager,mail.activity.type.sale.manager,mail.model_mail_activity_type,sales_team.group_sale_manager,1,1,1,1
+access_lead_scoring_frequency,access_lead_scoring_frequency,model_lead_scoring_frequency,sales_team.group_sale_salesman,1,1,1,1

```

Comment:
Not sure it should be that way.
'Salesman' users will only need to read that model to get the probability right?

It's probably because they can also "rebuild" the frequencies table with a button. 
But then the button action should sudo the model behind the scenes instead IMO.
With those security rules, salesman can use xmlrpc to add/update/delete the frequencies table, and I guess it's not the point.

Maybe I don't fully understand this though but please double check.